packages/flutter_tools/bin/podhelper.rb

```diff
@@ -73,9 +73,13 @@ def flutter_additional_ios_build_settings(target)
 
 # Same as flutter_ios_podfile_setup for macOS.
 def flutter_additional_macos_build_settings(target)
-  print target.platform_name
   return unless target.platform_name == :osx
 
+  # [target.deployment_target] is a [String] formatted as "10.8". Check if this is < 10.9
+  # See MACOSX_DEPLOYMENT_TARGET note below.
+  deployment_target_major, deployment_target_minor = target.deployment_target.match(/(\d*).?(\d*)/).captures
+  inherit_deployment_target = deployment_target_major.to_i < 10 || deployment_target_minor.to_i < 9
```

Comment:
This logic isn't correct; it treats 11.[0-8] as older. Would be good to add an 11.0 test case that would have caught that.

Reply:
Well I failed the interview.